src/Message/DirectAuthorizeRequest.php

```diff
@@ -56,6 +56,30 @@ protected function getBaseAuthorizeData()
         $data['VendorTxCode'] = $this->getTransactionId();
         $data['ClientIPAddress'] = $this->getClientIp();
 
+        $data['BrowserJavascriptEnabled'] = $this->getBrowserJavascriptEnabled() ?: static::BROWSER_JAVASCRIPT_NO;
+        $data['BrowserLanguage'] = $this->getBrowserLanguage() ?: static::BROWSER_LANGUAGE;
+        $data['ThreeDSNotificationURL'] = $this->getThreeDSNotificationURL();
+        $data['BrowserAcceptHeader'] = $_SERVER['HTTP_ACCEPT'];
```

Comment:
I would just add  `?? null`

Reply:
Have done this, errors relating to these lines weren't showing when running the tests locally.